src/main/java/net/syscon/elite/repository/jpa/model/LivingUnit.java

```diff
@@ -0,0 +1,157 @@
+package net.syscon.elite.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinColumnsOrFormulas;
+import org.hibernate.annotations.JoinFormula;
+import org.hibernate.annotations.NotFound;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.time.LocalDate;
+import static org.hibernate.annotations.NotFoundAction.IGNORE;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@IdClass(LivingUnit.PK.class)
+@Table(name = "LIVING_UNITS")
+public class LivingUnit {
+
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class PK implements Serializable {
+
+        @Column(name = "LIVING_UNIT_ID", insertable = false, updatable = false, nullable = false)
+        private Long livingUnitId;
+
+        @Column(name = "DESCRIPTION", nullable = false)
+        private String description;
+
+        @Column(name = "AGY_LOC_ID", nullable = false)
```

Comment:
We have the models in place to support this already:

```
    @ManyToOne(optional = false)
    @JoinColumn(name = "AGY_LOC_ID", nullable = false)
    private AgencyLocation agencyLocationId;

```

Reply:
In this case the agency location column is part of the composite primary key, so I don't know if it's possible to use the model instead. Also don't need the agency information as part of the object, purely used for filtering.

Reply:
Just realised I put this on the wrong line anyway. Meant for line 53 which I think would be OK.  Not matter.